layers/+tools/restclient/packages.el

```diff
@@ -10,9 +10,12 @@
 ;;; License: GPLv3
 (setq restclient-packages
       '(
+        company
+        company-restclient
         golden-ratio
         ob-http
         restclient
+        restclient-helm
```

Comment:
Is there a meaningful difference between `layer-usedp` and `package-usedp` in this context? Seems like the latter is used more often, but the distinction escapes me.